client/js/map/index.js

```diff
@@ -8,6 +8,7 @@ import { siteBoundaryHelp } from './markUpItems.js'
 import { vtLayers } from './vtLayers.js'
 import { setUpBaseMaps } from './baseMap.js'
 import { encode } from '@mapbox/polyline'
+import { decodePolygon } from '../../../server/services/shape-utils.js'
```

Comment:
remember that we were going to put encode in shape utils too - so it can be swapped with a different method easily.